src/context/AppProvider/AppContext.tsx

```diff
@@ -153,7 +156,10 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
               const bitcoin = assetsById[CAIP19]
 
               if (!bitcoin) continue
-              for (const accountType of supportedAccountTypes.bitcoin) {
+              const supportedAccountTypes = (
+                adapter as UTXOBaseAdapter<UTXOChainTypes>
```

Comment:
If we don't cast it, this is going to give us some tsc errors about calling a possibly undefined, non-callable method. It's either that or using the `?.` method access notation and casting the result 🤷🏽